frontend/package.json

```diff
@@ -23,6 +23,7 @@
     "@emotion/css": "^11.10.0",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.10.0",
+    "@iconify-icons/ep": "^1.2.12",
```

Comment:
lets try to stick to `tabler/icons-react` or decide on a single set of icons to use in general, i think we have too many in the project now 😆